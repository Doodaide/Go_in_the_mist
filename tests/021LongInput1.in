who
place D11
who
place M14
place I16
place O10
place D14
place P13
history
view
termmmm
who
place D12
place K4
place D13
place O2
place D10 
place d100
place D10