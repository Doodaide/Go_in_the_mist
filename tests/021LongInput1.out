B
W
D11M14I16O10D14P13
C15,x......x......x......x......x...#..x......x......
Invalid!
B
Invalid!
Invalid coordinate
Black wins!
D11M14I16O10D14P13D12K4D13O2D10
Thank you for playing!
